modules/custom/az_paragraphs/az_paragraphs_text_media/templates/paragraph--az-text-media--default.html.twig

```diff
@@ -79,7 +79,7 @@
   ]
 %}
 {% block paragraph %}
-  <div{{ attributes.addClass(classes).setAttribute('id', paragraph.bundle() ~ '-' ~ paragraph.id()) }}>
```

Comment:
Remove setting the ID from the template file and moved that functionality here: https://github.com/az-digital/az_quickstart/pull/1042/files#diff-3835e62d0ade8930c6536b446dfe773d6022f3eee035e46dbee2b12ebcbfb8afR172